docs/buildrun.md

```diff
@@ -154,18 +154,17 @@ See more about _paramValues_ usage in the related [Build](./build.md#defining-pa
 A `BuildRun` resource can define a serviceaccount to use. Usually this SA will host all related secrets referenced on the `Build` resource, for example:
 
 ```yaml
-apiVersion: shipwright.io/v1alpha1
+apiVersion: shipwright.io/v1beta1
 kind: BuildRun
 metadata:
   name: buildpack-nodejs-buildrun-namespaced
 spec:
-  buildRef:
+  build:
     name: buildpack-nodejs-build-namespaced
-  serviceAccount:
-    name: pipeline
+  serviceAccount: pipeline
 ```
 
-You can also use set the `spec.serviceAccount.generate` path to `true`. This will generate the service account during runtime for you. The name of the generated service account is the name of the BuildRun. **This field is deprecated, and will be removed in a future release.**
+You can also set the value of `spec.serviceAccount` to `".generate"`. This will generate the service account during runtime for you. The name of the generated service account is the name of the BuildRun.
```

Comment:
```suggestion
You can also set the value of `spec.serviceAccount` to `".generate"`. This will generate the service account during runtime for you. The name of the generated service account is the same as that of the BuildRun.
```